examples/connectivity/plot_mixed_source_space_connectity.py

```diff
@@ -58,8 +58,9 @@
                          spacing='oct6', add_dist=False, overwrite=True)
 
 # Setup a volume source space
+# set pos=7.0 for computational issue
```

Comment:
better write "for speed", otherwise people reading the example may think there is some issue with computation and all will be setting pos=0.7 every time from the fear of unknown. :)